charts/ai-service/templates/deployment.yaml

```diff
@@ -188,4 +154,20 @@ spec:
               secretProviderClass: {{ $fullName }}-{{ $k }}
       {{- end }}
       {{- end }}
+      {{- if .Values.artifactsCache.enabled }}
+      initContainers:
```

Comment:
This is probably how we want how initContainers to look across charts https://github.com/Unique-AG/helm-charts/blob/f133a7a375e263b46c4e76b5f44f461de5a3a2e6/charts/backend-service/templates/deployment.yaml#L64-L78